app/pods/publications/publication/controller.js

```diff
@@ -27,51 +38,58 @@ export default class PublicationController extends Controller {
       .toUpperCase();
   }
 
+  // TODO get status
+  // TODO set selected item from the status value
+
   get getPublicationNumberSidePanel() {
-    console.log('P number', this.model.get('publicationNumber'));
     if (this.model.get('publicationNumber')) {
       this.publicationNumber = this.model.get('publicationNumber');
     }
     return this.publicationNumber;
```

Comment:
Waarom willen we de properties hier ook nog eens tracked maken? We zouden dit toch rechtstreeks van het model kunnen halen en gewoon rechtstreeks `this.model.get('publicationNumber');` doen?